templates/common/infra/bicep/core/storage/storage-account.bicep

```diff
@@ -21,6 +22,17 @@ resource storage 'Microsoft.Storage/storageAccounts@2022-05-01' = {
       defaultAction: 'Allow'
     }
   }
+
+  resource blobServices 'blobServices' = if (!empty(containers)) {
+    name: 'default'
+    resource container 'containers' = [for container in containers: {
+      name: container.name
+      properties: {
+        publicAccess: container.publicAccess
```

Comment:
Should we have a check here whether the `publicAccess` property is set on the container? I would expect this might fail if it is `null` or a non-boolean value?

Reply:
Good call.  Thanks.  I updated to check for existence o fthe prop and to default to 'None' if not found.